functors_task_1/student/src/impl.cpp

```diff
@@ -1,2 +1,29 @@
 #include "impl.h"
 
+BeerBrand BeerOrganizer::operator()() {
+    if (static_cast<size_t>(BeerBrand::Max) == currentIndex)
+        currentIndex = 1;
```

Comment:
avoid magic numbers, specify "from" and "to" const variables

Reply:
Changed to use enum values